mne/decoding/tests/test_transformer.py

```diff
@@ -160,3 +160,13 @@ def test_epochs_vectorizer():
     # Test init exception
     assert_raises(ValueError, vector.fit, epochs, y)
     assert_raises(ValueError, vector.transform, epochs, y)
+
+
+def test_vectorizer():
+    """Test Vectorizer."""
+    data = np.random.rand(150, 18, 6)
+    vect = Vectorizer()
+    result = vect.fit_transform(data)
+    assert_equal(result.ndim, 2)
+    # check inverse_trasnform
+    assert_equal(vect.inverse_transform(result).ndim, 3)
```

Comment:
Additionally, check that you can't pass an array with a different number of features with an `assert_raise`.